src/plone/restapi/pas/plugin.py

```diff
@@ -39,7 +41,13 @@ def addJWTAuthenticationPlugin(self, id_, title=None, REQUEST=None):
         )
 
 
-@implementer(IAuthenticationPlugin, IChallengePlugin, IExtractionPlugin)
+@implementer(
+    IAuthenticationPlugin,
+    IChallengePlugin,
+    IExtractionPlugin,
+    plugins_ifaces.ICredentialsUpdatePlugin,
```

Comment:
I think the already existing style of imports should be kept.

Reply:
I find the `module.attribute` form of reference to imported code to be much more readable because I can see where the imported code comes from where it's code is used.  But not a hill I'll die on today.  Done.